src/main/java/org/apache/sling/api/resource/ResourceResolver.java

```diff
@@ -842,4 +842,20 @@ Resource copy(final String srcAbsPath,
      */
     Resource move(final String srcAbsPath,
               final String destAbsPath) throws PersistenceException;
+
+    /**
+     * Returns a map to store temporary objects.
+     *
+     * This map is suited to store objects which share the same lifecycle as the
+     * resource resolver. The resource resolver itself does not use this map.
+     *
+     * The resource resolver will clear the map during {@link #close()}, so afterwards the map is empty.
```

Comment:
Close() may throw exceptions. Are those propagated to the close method of the RR? The javadoc should be clarified in that regard.

Reply:
Good point, I would assume they get ignored

Reply:
updated javadocs